openapi.yaml

```diff
@@ -31,71 +25,69 @@ paths:
       operationId: postPerson
-      parameters:
-        - name: personBody
-          in: body
-          required: true
-          schema:
-            $ref: '#/definitions/PersonPostBody'
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/PersonPostBody'
+        required: true
       responses:
         '202':
-          description: The backend data source needs to do additional processing 
-            with the new person record. Person record accepted but not yet created.
-          schema:
-            $ref: '#/definitions/PersonResult'
+          description: The backend data source needs to do additional processing with the
+            new person record. Person record accepted but not yet created.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/PersonResult'
         '400':
-          $ref: '#/responses/400'
+          $ref: '#/components/responses/400'
         '409':
-          $ref: '#/responses/409'
+          $ref: '#/components/responses/409'
         '500':
-          $ref: '#/responses/500'
+          $ref: '#/components/responses/500'
   /persons/{osuId}:
     get:
       description: Get general person information
       tags:
         - persons
       operationId: getPersonById
       parameters:
-        - $ref: '#/parameters/osuId'
+        - $ref: '#/components/parameters/osuId'
       responses:
         '200':
           description: Successful response
-          schema:
-            $ref: '#/definitions/PersonResult'
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/PersonResult'
         '404':
-          $ref: '#/responses/404'
+          $ref: '#/components/responses/404'
         '500':
-          $ref: '#/responses/500'
+          $ref: '#/components/responses/500'
   /persons/{osuId}/jobs:
     get:
       description: Get jobs associated with a person
       tags:
         - jobs
       operationId: getJobs
       parameters:
-        - $ref: '#/parameters/osuId'
+        - $ref: '#/components/parameters/osuId'
         - name: filter[positionNumber]
           in: query
-          type: string
           required: false
           description: Filter by position ID.
+          schema:
+            type: string
         - name: filter[suffix]
           in: query
-          type: string
           required: false
           description: >
             Filter by suffix. The suffix is used to differentiate multiple jobs
             that share the same position number for a given person.
+          schema:
+            type: string
         - name: filter[statusCode]
           in: query
-          type: string
-          enum: &jobStatusEnums
-            - A
-            - B
-            - L
-            - F
-            - P
-            - T
           required: false
           description: |
```

Comment:
Seems like you use both `>` and `|`. I guess we should use `>` for most of the cases. Can you check the code and make them consistent and also create a PR in style guide for this?

Reply:
`>` strips linebreaks while `|` does not so they have different uses. I can still add this to the style guide.